README.md

```diff
@@ -9,7 +9,7 @@ Apache StormCrawler (Incubating) is an open source collection of resources for b
 
 ## Quickstart
 
-NOTE: These instructions assume that you have [Apache Maven](https://maven.apache.org/install.html) installed. You will need to install [Apache Storm 2.6.2](http://storm.apache.org/) to run the crawler.
+NOTE: These instructions assume that you have [Apache Maven](https://maven.apache.org/install.html) installed. You will need to install [Apache Storm 3.1.1](http://storm.apache.org/) to run the crawler.
```

Comment:
This needs to be 2.6.4.

The 3.1.0 needs to be in the archetype bash example

Reply:
Ok got that! I will change it !